vagrant/Vagrantfile

```diff
@@ -96,7 +96,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       }
     }
     chef.run_list = [
-      "recipe[aet::_develop]",
+#      "recipe[aet::_develop]",
```

Comment:
Thank you for this PR @wiiitek :)
I think we should use `aet::_develop` by default on Vagrant - it creates `developer` user which allows us to easily upload AET to Vagrant with maven using `developer/developer` credentials, as instructed in [AET in 10 minutes](https://github.com/Cognifide/aet/wiki/AETIn10Minutes#upload-vagrant-prequisities)

Reply:
You're right! This change was to test the use case described in https://github.com/Cognifide/aet-cookbook/pull/30#discussion_r212208661. We should probably do a release for the aet-cookbook (version 5.0.0). What do you think?